src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -217,13 +217,13 @@
     <value>Add a member to {0} that has a value of zero with a suggested name of 'None'.</value>
   </data>
   <data name="AbstractTypesShouldNotHaveConstructorsTitle" xml:space="preserve">
-    <value>Abstract types should not have constructors</value>
+    <value>Abstract types should not have public constructors</value>
   </data>
   <data name="AbstractTypesShouldNotHaveConstructorsDescription" xml:space="preserve">
     <value>Constructors on abstract types can be called only by derived types. Because public constructors create instances of a type, and you cannot create instances of an abstract type, an abstract type that has a public constructor is incorrectly designed.</value>
   </data>
   <data name="AbstractTypesShouldNotHaveConstructorsMessage" xml:space="preserve">
-    <value>Abstract type {0} should not have constructors</value>
+    <value>Abstract type '{0}' should not have public constructors</value>
```

Comment:
I haved also added the missing quotes, let me know if you prefer them not to be set in this PR.